Gruntfile.js

```diff
@@ -108,41 +108,9 @@ module.exports = function(grunt) {
       }
     },
     jscs: {
-      files: {
-        src: [
-          "Gruntfile.js",
-          "lib/**/!(johnny-five)*.js",
-          "test/**/*.js",
-          "eg/**/*.js",
-        ]
-      },
+      src: "**/*.js",
```

Comment:
I have a local directory called `wip` that's full of junk files, experiments, copy/paste crap, etc. I don't want those files to be reviewed by jscs, as they will most certainly cause false failures locally.